examples/sample-app/README.md

```diff
@@ -349,9 +349,18 @@ the ip address shown below with the correct one for your environment.
             # Optional: pre-pull the router image.  This will be pulled automatically when the pod is created but will
             # take some time.  Your pod will stay in Pending state while the pull is completed
             $ docker pull openshift/origin-haproxy-router
+            
+            # Create a service account that the router will use.  This service account must have access to use a
+            # security context constraint that allows host ports
+            $ echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"router"}}' | oc create -f -
+            
+            # Edit your security context constraint to add the new service account in the users section
```

Comment:
How can I know whether a given SCC has sufficient privileges?  You described the SCC required in the paragraph above, but didn't provide details to tell me how I can tell whether my SCCs are sufficient.  If we have a reasonable, enumerated list, it might be reasonable to have a command that can perform the "which SCCs are good enough" test.

For now, I think you should provide an example of SCC permissions that are sufficiently broad.  Otherwise, I'm not sure how we expected a user to even know how to select or create one.